### ISSUE TITLE
last plot doesn't run

### ISSUE THREAD

**Issue by mlayvant** [StoneYan1332/Mexico#4]
There is a key error in the graph that is supposed to show changes to GDP over time, so the plot isn't running.

**Comment by StoneYan1332** [comment]
Yep. The column for date wasn't indexed, so we used reset_index to make it callable by the data frame. Now it runs fine. Thank you for pointing that out!
